Jenkinsfile.macoslinux.crossbuild

```diff
@@ -189,6 +191,8 @@ image.inside("-w /opt/browser-f"){
     stage('Unstash signed artifacts'){
         unstash 'signeddedmg'
```

Comment:
it does  not seem we still use DMGs, this can be removed to speed up the build

Reply:
Yes, this is my thoughts also, I will switch to use only apps

Reply:
Umm. I tried to play with this idea - it's hard, because processing DMG is not native on Linux platform, so for packing-unpacking I need to investigate DMG_TOOL, MKFSHFS and HFS_TOOL tools. It absolutely worth to learn them and switch to using it, but now I need to make quick solution for new version